services/venice-controller/src/main/java/com/linkedin/venice/controller/kafka/consumer/AdminConsumptionTask.java

```diff
@@ -828,6 +829,10 @@ private String extractStoreName(AdminOperation adminOperation) {
                   + " because it does not contain a storeName field");
         }
     }
+    VeniceSystemStoreType systemStoreType = VeniceSystemStoreType.getSystemStoreType(storeName);
```

Comment:
can we add this as an util method, so many places does this regular store name extraction ?

Reply:
Will do